openshift-kube-apiserver/openshiftkubeapiserver/flags.go

```diff
@@ -42,6 +46,15 @@ func ConfigToFlags(kubeAPIServerConfig *kubecontrolplanev1.KubeAPIServerConfig)
 	configflags.SetIfUnset(args, "tls-min-version", kubeAPIServerConfig.ServingInfo.MinTLSVersion)
 	configflags.SetIfUnset(args, "tls-sni-cert-key", sniCertKeys(kubeAPIServerConfig.ServingInfo.NamedCertificates)...)
 
+	// set the global featuregates.
```

Comment:
This piece of code should not be necessary, once you address the other comment.

Reply:
Ok, I have removed it. This was indeed redundant, I tested without this block of code, and can still get:

```diff
I0314 20:34:51.747391  204157 feature_gate.go:250] feature gates: &{map[
CloudDualStackNodeIPs:true DisableKubeletCloudCredentialProviders:false 
DynamicResourceAllocation:false EventedPLEG:false KMSv1:true 
MaxUnavailableStatefulSet:false NodeSwap:false 
+ RouteExternalCertificate:true
TranslateStreamCloseWebsocketRequests:false ValidatingAdmissionPolicy:false]}
```
which helped clarify.